content/api/transmissions.apib

```diff
@@ -16,7 +16,7 @@ Learn about [how to optimize your sending](https://www.sparkpost.com/docs/tech-r
 
 ### The Sandbox Domain
 
-The sandbox domain allows you to send your first email without setting up a sending domain. You can send up to 5 sandbox messages over the lifetime of your account. To send a message from the sandbox domain, set the `from` address to `anything@sparkpostbox.com` and set `options.sandbox` to `true`. Just like with any sending domain, you can set the part before the `@` to any mailbox.
+The sandbox domain allows you to send your first email without setting up a sending domain. You can send up to 5 sandbox messages over the lifetime of your account. To send a message from the sandbox domain, set `options.sandbox` to `true`, and set `content.template_id` to `my-first-email`. The `from` address in the template `my-first-email` should be `anything@sparkpostbox.com`. Just like with any sending domain, you can set the part before the `@` to any mailbox. You can only use the sandbox domain with the `my-first-email` template.
```

Comment:
The from address in the my-first-email template is support@sparkpostbox.com.

Reply:
Do we want to remove this line as well then 
since this template is auto-generated
```
Just like with any sending domain, you can set the part before the `@` to any mailbox.

Reply:
Yes, I think that we should